source/_posts/2020-09-18-mac-companion.markdown

```diff
@@ -0,0 +1,63 @@
+---
+title: "Home Assistant Companion for macOS"
```

Comment:
based on an idea from @ludeeus:

One more thing...

Reply:
As in change the title to "One more thing..."? I like it

Reply:
Yeah, that as the title was the idea

Reply:
Updated and also the description to now give context but also be in the "One more thing..." style

Reply:
Nice :tada:  :smile: